ElectronicObserver/Window/Control/Paging/PagingControlViewModel.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+
+namespace ElectronicObserver.Window.Control.Paging;
+
+public partial class PagingControlViewModel : ObservableObject
+{
+	public int CurrentPage { get; set; } = 0;
+
+	private ObservableCollection<object?> Items { get; set; } = new();
```

Comment:
`set` is never used.

Reply:
Why is the generic parameter `object?`?
I can't really think of an example where null items in a collection would be useful.
Also update the other collections to disallow null.

Reply:
I brought back the generic type I was trying to use before initial commit.
Easier to use, but has to be typed as an object in `PagingControlView`.

`set` stays public and is now used to set the collection of the pager instead of using `public void UpdateSourceCollection<T>(IList<T> collection)` and the `public PagingControlViewModel(ObservableCollection<object?> items)` constructor